arches/app/datatypes/datatypes.py

```diff
@@ -1712,30 +1712,33 @@ def pre_tile_save(self, tile, nodeid):
                     if ("resourceXresourceId" not in related_resource or related_resource["resourceXresourceId"] == "")
                     else related_resource["resourceXresourceId"]
                 )
+                related_resourceid = related_resource["resourceId"]
                 defaults = {
                     "resourceinstanceidfrom_id": tile.resourceinstance_id,
-                    "resourceinstanceidto_id": related_resource["resourceId"],
+                    "resourceinstanceidto_id": related_resourceid,
                     "notes": "",
-                    "relationshiptype": related_resource["ontologyProperty"],
-                    "inverserelationshiptype": related_resource["inverseOntologyProperty"],
                     "tileid_id": tile.pk,
                     "nodeid_id": nodeid,
                 }
-                if related_resource["ontologyProperty"] == "" or related_resource["inverseOntologyProperty"] == "":
-                    if models.ResourceInstance.objects.filter(pk=related_resource["resourceId"]).exists():
-                        target_graphid = str(models.ResourceInstance.objects.get(pk=related_resource["resourceId"]).graph_id)
+                if "ontologyProperty" not in related_resource or "inverseOntologyProperty" not in related_resource:
+                    if models.ResourceInstance.objects.filter(pk=related_resourceid).exists():
```

Comment:
:-1: to this check.  This would break a batch load with circular references.  Just load them and let it become eventually consistent.